### PR TITLE
[DPE-2765] Add `timescaledb` package

### PR DESCRIPTION
**Pull request by TakoB222**
### Issue
Add packed to support `timescaledb` plugin/extension

> ## Solution
Added `timescaledb-2-postgresql-14` package which contains `timescaledb` plugin/extension

##
Building time:
 5m04.77s # before the PR
 6m01.25s # after the PR

Snap size:
-rw-r--r-- 1 yevs yevs 181186560 Feb 15 14:27 charmed-postgresql_14.10_amd64.snap # before the PR
-rw-r--r-- 1 yevs yevs 266682368 Feb 15 14:36 charmed-postgresql_14.10_amd64.snap # after the PR

### CODE REVIEW

**Comment by taurus-forever** [comment]
Hi @TakoB222 , thank you for the contribution!
@delgod , @7annaba3l : can we use foreign timescaledb binary? Should we PPA it first?

**Comment by taurus-forever** [comment]
Just for the history, we are waiting for timescaledb package built in Canonical. (c) @7annaba3l
Update: in the same time @TakoB222 is trying to rebuild it from his side to speedup the process. (c) @delgod

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -40,6 +40,8 @@ package-repositories:
     ppa: data-platform/python-pysyncobj
   - type: apt
     ppa: data-platform/pgbackrest-exporter
+  - type: apt
+    ppa: yevss/postgresql-14-timescaledb
```
Comment (taurus-forever) [inline snap/snapcraft.yaml]: Copied here https://launchpad.net/~data-platform/+archive/ubuntu/postgresql-14-timescaledb
Please update snap config to build test snap.
We are still reviewing the package, please do NOT merge till the further notice. Tnx!

**Review by taurus-forever — CHANGES_REQUESTED** [review]

File: snap/snapcraft.yaml (hunk 1 above)
Comment (TakoB222) [reply snap/snapcraft.yaml]: @taurus-forever Updated PPA to fix failed build, I think it should be copied again

**Review by marceloneppel — APPROVED** [review]
LGTM! I tested it on the VM charm, and it works fine. Thanks, @TakoB222! Now, just wait for the confirmations that Alex commented on.

**Comment by taurus-forever** [comment]
@TakoB222 we have updated timescaledb package with verified content and licenses OSI compliance (built with `-DAPACHE_ONLY=1`).
Please rebuild snap and re-check it to merge PR.

https://launchpad.net/~data-platform/+archive/ubuntu/postgresql-14-timescaledb

Tnx!

File: snap/snapcraft.yaml (hunk 1 above)
Comment (taurus-forever) [reply snap/snapcraft.yaml]: I am lost here. Why are you using `yevss` PPA and now `data-platform` PPA directly?

Comment (TakoB222) [reply snap/snapcraft.yaml]: @taurus-forever I changed build details for `yevss` PPA and deployed it to test

**Comment by TakoB222** [comment]
> @TakoB222 we have updated timescaledb package with verified content and licenses OSI compliance (built with `-DAPACHE_ONLY=1`). Please rebuild snap and re-check it to merge PR.
> 
> https://launchpad.net/~data-platform/+archive/ubuntu/postgresql-14-timescaledb
> 
> Tnx!

Done

**Review by marceloneppel — APPROVED** [review]
Thanks @TakoB222! LGTM!

**Review by dragomirp — APPROVED** [review]

**Review by taurus-forever — APPROVED** [review]
Thank you!

**Pull request merged**